downloads/platform.md

```diff
@@ -172,3 +172,20 @@ Julia is available in the [Ports Collection](https://svnweb.freebsd.org/ports/he
 ```
 pkg install julia
 ```
+
+## Arch Linux
+
+Julia can be installed with
+
+```
+sudo pacman -S julia
```

Comment:
Why is that?

Reply:
Because the `julia` package uses system libraries.  Julia is quite picky about the libraries to link to and the official binaries provide all the necessary ones by default, `aur/julia-bin` uses these official binaries.

Reply:
I see. I hope Julia will become less picky as it matures. Thanks!

Reply:
Recent example of user struggling with the `julia` package: https://github.com/JuliaLinearAlgebra/Arpack.jl/issues/97.  There are also frequent reports of people having problems with the system libgit2 library, and hence the Julia `Pkg` package

Reply:
Right, I indeed got the libgit2 issue. ;) As using system libraries is a feature, I think it would be helpful if distro packages for julia could be made to work. I appreciate packaging is difficult, but many users will want and expect their packages to work.

Reply:
It is not difficult. Many libraries have bugs and need patches we apply - which is easily managed by adopting those patches. The main issue is that many distributions want you to use system llvm which is often just plain broken for Julia.

Reply:
I get that and am thankful that you're fixing those bugs—and hopefully trying to upstream them. To me, using system llvm is a feature: It makes for an integrated system. Llvm however seems to be a difficult dependency to work with (you're not alone having issues).

Reply:
Yes, we upstream all our work. And distros apply several patches themselves to llvm too. So, we don't know if those will affect Julia in any way. There really isn't a good immediate solution to this - but the one thing we can do is ensure our distribution does all the right things. It also reduces variability from system to system and improves reproducibility.